contents/docs/product-analytics/correlation.mdx

```diff
@@ -10,52 +10,40 @@ availability:
 
-We can drill down further to see how this events properties are correlated as well. Click on the expand event button to the left and we see that people that opted for the redesign and did not have a billing plan were 3.3x times less likely to convert.
-
-Where we go from here requires further investigation, but the important point is that we now have strings to pull on which hopefully end up with actionable insights.
-
-## What to do with this newfound knowledge?
-
-One route for further exploration might be to drill down to the people that were considered for this specific event correlation. We can do so by clicking on the "Completed" and "Dropped off" numbers within a row. In the above case, users for both "Completed" and "Dropped off" will have performed the "opted into navigation redesign". "Completed" means the users fully completed the funnel, where as "Dropped off" users will include users who entered the funnel, but failed to make it through to the end.
-
-Another option might be to take, for instance, the "Country Name" property and add this as a funnel breakdown to further explore and understand the affect it has on the funnel:
-
-![Breakdown by country](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/docs/user-guides/funnels/funnel-correlation-analysis-country.png)
-
-## Making correlations accurate
-
-You have better context on your product, and what could be a relevant signal or not. There will be some events and/or properties that perhaps have trivial correlations, or don't actually provide an actionable signal. To remove this
-potential distractions, use either the property names drop down to ignore them in analysis for a funnel in particular, or use the _Exclude from project_ feature which will ensure that you and your team will not see correlations for these events and properties whilst within this project.
-
-It's important to remember that the accuracy and therefore usefulness of any correlation is dependent on the sample size, if you've only got a handful of users for a specific correlation then it would be wise to try to increase this number. You can do this by, for instance, increasing the date range size of the funnel.
+<ProductScreenshot
+    imageLight = {"https://res.cloudinary.com/dmukukwp6/image/upload/v1716389459/posthog.com/contents/Screenshot_2024-05-22_at_3.50.44_PM.png"} 
+    imageDark = {"https://res.cloudinary.com/dmukukwp6/image/upload/v1716389459/posthog.com/contents/Screenshot_2024-05-22_at_3.50.00_PM.png"}
+    classes="rounded"
+    alt= "How to hide correlations"
+/>
 
 ### Further reading
 
-Want to know more about what's possible with Correlation in PostHog? Try these tutorials:
-
--   [How to correlate Sentry errors with PostHog data](/tutorials/sentry-plugin-tutorial)
-
-Want more? Check our [full list of PostHog tutorials](https://posthog.com/tutorials).
+- [Tutorial: How to correlate Sentry errors with PostHog data](/tutorials/sentry-plugin-tutorial)
```

Comment:
This tutorial doesn't really have to do much with correlation analysis, it just has "correlate" in its name.
```suggestion
```